extensions/amp-auto-ads/0.1/test/test-ad-network-config.js

```diff
@@ -109,6 +109,7 @@ describes.realWin('ad-network-config', {
       expect(adNetwork.getAttributes()).to.deep.equal({
         'type': 'adsense',
         'data-ad-client': 'ca-pub-1234',
+        'data-ad-host': null,
```

Comment:
Please also add a new test case where the data-ad-host attribute is set on the ampAutoAdsElem.